### PR TITLE
Update base font size to 14px and tweak heading sizes accordingly

### PR DESCRIPTION
**Pull request by deioo**
What says on the tin.

### CODE REVIEW

**Comment by matteodepalo** [comment]
Could you explain briefly why the rem-base has to be different from the base font-size? I've seen this before and it left me a bit perplexed (because of my ignorance in the subject). I've seen Foundation sets the base font size to 100% and then changes the rem-base to a precise value in pixels.

**Comment by deioo** [comment]
While em is relative to the font-size of its direct or nearest parent, rem is only relative to the html (root) font-size.

So in our case base font size (root) is set to 14px. 
Rem-base is the base unit used for rem calculations. By default it's 16px (100%), so 1rem is interpreted as 16px. 
If we've changed the base font size to 14px, then the rem base would become 14px and all rem-calc values would become 0.875 of what they actually are.
That's why we're forcing it to 16px, so we don't get weird pixel values when computing spacing and font sizes in general.

File: globals/_settings.scss [hunk 1]
```diff
@@ -47,12 +47,12 @@ $header-font-family: "Helvetica Neue", Helvetica, sans-serif;
...
-$h1-font-size: 2.375rem;
+$h1-font-size: 2.25rem;
 $h2-font-size: 1.75rem;
-$h3-font-size: 1.3125rem;
-$h4-font-size: 1rem;
-$h5-font-size: 0.875rem;
-$h6-font-size: 0.75rem;
+$h3-font-size: 1.375rem;
+$h4-font-size: 1.125rem;
+$h5-font-size: 1rem;
```
Comment (deioo) [inline globals/_settings.scss]: As related to previous comment, this outputs to 16px, while h6 outputs to 14px.

**Comment by matteodepalo** [comment]
:+1: thanks for explaining.

**Pull request merged**